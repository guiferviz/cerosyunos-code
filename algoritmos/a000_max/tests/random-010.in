40
8
492884559 1609957982 1017285151 377026458 1355543569 708873758 1419830988 334109034
8
985042574 1854142046 1598136938 532107822 1661396981 1713520762 447984730 394775362
8
488930683 1249716503 211347699 1960759564 794715427 1035271743 937869870 1376230299
8
1025267859 1836424634 363294727 441512839 1205832539 1655271404 204626465 1262621375
8
452219619 420832845 1388060578 1115009681 1199819629 1822348593 1436711489 2027782275
8
1220216638 1929884140 363722131 1405563406 1178258699 484356020 398417852 1589615801
8
215405814 1720975521 1657294190 1571020954 1021896847 742516525 389964467 745655688
8
1048585900 1041675590 1545727424 1103581686 1081043691 273946145 197128131 1116611099
8
1543201641 1487763073 726233663 922368185 2074256639 2013170866 420986442 1871595430
8
1464084883 519964786 42946611 615542539 1802582912 1770949666 1283832370 1639749638
8
403073106 1596249514 603990261 464193143 783869883 714531276 603858470 286115967
8
1752987627 1628223285 219948663 75644336 520144168 1181610965 1490056521 244983617
8
1101642662 1247754367 527026575 743780225 449705789 442936002 295327588 1635876543
8
1415007300 2049151287 1929122004 1412308275 633372875 168414786 1638455262 2073206976
8
1840154526 910096250 292572125 1608454851 513624281 432120069 1037747974 1011830125
8
1727288297 1416658200 1922596782 2112764748 511206245 918192805 1078231496 407254595
8
1220305354 162892249 938534543 1579795986 389693508 2097538591 1846318681 1681974036
8
61368711 1938642971 1705061477 894128006 1853454041 935331340 634794968 1960094576
8
294409710 156419653 1772460572 1014222157 1836689942 983262414 984469998 193017785
8
1273454906 1538430484 465252988 625774272 1763215831 2104418080 1089488239 1347879662
8
1047347934 671343271 1954639843 944613855 1194813998 577811789 1775623133 869393505
8
482926785 764538000 144985317 1903940450 749234939 748332011 661442922 1652754713
8
189881491 6454974 92390910 369554697 476180975 339575077 703684813 818708798
8
320029893 1224466735 2098391620 1488589684 1203189472 1324757882 1259522729 2030702336
8
116966222 165700627 847628233 1764409795 1146324770 591020360 1538236504 1906431628
8
1237977599 831652059 1884121011 181572535 1303334628 1611296615 1318128582 1199987941
8
181078777 1443241361 409647670 1614687460 1578577671 2131662592 1319310542 1361085652
8
1650842679 978241814 2083100628 967080748 1086000648 974364477 1534425556 213252784
8
249026717 1370913694 902285935 725920772 1258676966 1133482644 470735315 869896379
8
1804182162 815972657 994126035 1459602489 658659556 498858740 1484115636 1922430204
8
1606278611 2016442910 887387886 388244893 811066704 1866055941 1554696739 1825134516
8
648846801 2049121087 1495168644 2119903260 857495434 1445706720 2111691074 1779833731
8
28571104 64893934 132130846 1749854681 1036410167 1217203624 1398037444 1154117014
8
1745569410 1705637688 398872226 1140818152 179453628 1422887184 1204065985 1351892507
8
1588186120 1368674568 264741261 303344121 1168307122 1833775437 528423822 1308804439
8
1799729965 1759713525 1727896978 698159268 1183349733 345740712 1694268587 194613905
8
931488533 133267293 997176584 1290749057 1103188254 1193529723 1967397312 1009177032
8
1057891857 1564428915 1634633297 1983663350 405047696 946895662 1859367644 321717643
8
383945943 1486521726 1455412780 351138602 893299507 588717059 833223367 2037431941
8
100965609 981662491 887014867 1166578189 480120150 2131933758 2084754759 1057752819
