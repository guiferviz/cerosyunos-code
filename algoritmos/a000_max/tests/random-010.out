1609957982
1854142046
1960759564
1836424634
2027782275
1929884140
1720975521
1545727424
2074256639
1802582912
1596249514
1752987627
1635876543
2073206976
1840154526
2112764748
2097538591
1960094576
1836689942
2104418080
1954639843
1903940450
818708798
2098391620
1906431628
1884121011
2131662592
2083100628
1370913694
1922430204
2016442910
2119903260
1749854681
1745569410
1833775437
1799729965
1967397312
1983663350
2037431941
2131933758
